README.md

```diff
@@ -30,3 +58,8 @@ corepack yarn build
 
 This command generates static content into the `build` directory and can be
 served using any static contents hosting service.
+
+---
+
+This website is built using [Docusaurus 2](https://docusaurus.io/), a modern
+static website generator.
```

Comment:
We might want to simply remove this – we're now using Docosaurus v3, and it seems pointless to put that (ever so small) maintenance burden on us when the information is easily findable in the `package.json`.
```suggestion

```